Templates/GraphicsGem/Template/Code/Include/${Name}/${Name}TypeIds.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+namespace ${Name}
+{
+    // System Component TypeIds
+    inline constexpr const char* ${Name}SystemComponentTypeId = "${SysCompClassId}";
+    inline constexpr const char* ${Name}EditorSystemComponentTypeId = "${EditorSysCompClassId}";
+
+    // Module derived classes TypeIds
+    inline constexpr const char* ${Name}ModuleInterfaceTypeId = "{E22C9D32-526E-481F-8440-29E489EEB349}";
```

Comment:
This needs to be replaced with `"{${Random_Uuid}}"`

The Uuid logic template logic in the [engine_template.py](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/engine_template.py#L709-L734) will need to be updated in the future to be updated replace actual Uuids in this file with the `"{${Random_Uuid}}"` placeholder